### PR TITLE
Wolf howls

### PR DESCRIPTION
**Pull request by Gamemechanicwow**
## 🍰 Pullrequest

Updated: Following creatures use incorrect sound/emote while OOC:

- Timber Wolf https://www.wowhead.com/classic/npc=69/
- Young Wolf https://www.wowhead.com/classic/npc=299/
- Coyote https://www.wowhead.com/classic/npc=834/
- Young Scavenger https://www.wowhead.com/classic/npc=1508/
- Ragged Scavenger https://www.wowhead.com/classic/npc=1509/
- Gray Forest Wolf https://www.wowhead.com/classic/npc=1922/
- Elder Crag Coyote https://www.wowhead.com/classic/npc=2729/
- Prairie Wolf https://www.wowhead.com/classic/npc=2958/
- Prairie Stalker https://www.wowhead.com/classic/npc=2959/
- Prairie Wolf Alpha https://www.wowhead.com/classic/npc=2960/
- Longtooth Howler https://www.wowhead.com/classic/npc=5287/

### Proof

- In comments.

### Issues

- (https://github.com/Wall-core/Everlook-Bugtracker/issues/240)

### How2Test

- None

### Todo / Checklist

- [X] None

### CODE REVIEW

**Comment by Daribon** [comment]
Please provide any form of evidence.

**Comment by FlagFlayer** [comment]
I think the best evidence is the fact that in Classic you don't hear wolf howls in Mulgore that are 400 yards away. Current behaviour is really really off, and this is a fix that is logical and consistent with how other sounds are processed.

**Comment by Gamemechanicwow** [comment]
I usually don't mind digging for evidence. But I am not going to subscribe to classic for this one :>

I tried viewing some youtube videos for Elwynn forest playthroughs but didn't hear any howls. 

I don't have any evidence except the logic that if an object makes a sound the sound should come from the object.
SMSG_PLAY_OBJECT_SOUND vs SMSG_PLAY_SOUND

If anyone has relevant sniffs I wouldn't mind digging thru them.

**Comment by Kittnz** [comment]
https://github.com/TrinityCore/TrinityCore/issues/20712
This is an ancient mangos bug. SMSG_PLAY_OBJECT_SOUND should be used.

File: sql/migrations/20221106005233_world.sql [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+INSERT INTO `migrations` VALUES ('20221106005233');
+-- Add your query below.
+
+-- Events list for Timber Wolf
+DELETE FROM `creature_ai_events` WHERE `creature_id`=69;
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (6901, 69, 0, 1, 0, 10, 1, 60000, 600000, 120000, 600000, 6901, 0, 0, 'Diseased Timber Wolf - Howl and Emote OOC');
+
+DELETE FROM `creature_ai_scripts` WHERE `id`=6901;
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `priority`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(6901, 0, 0, 16, 1018, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Timber Wolf - Play Sound 1018'),
```
Comment (ratkosrb) [inline sql/migrations/20221106005233_world.sql]: No results in the mass parse classic sniff db for:
```
SELECT *
FROM sound_unique_source
WHERE source_id=69
```

File: sql/migrations/20221106005233_world.sql [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+(6901, 0, 0, 1, 15, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Timber Wolf - Emote 15');
```
Comment (ratkosrb) [inline sql/migrations/20221106005233_world.sql]: No results in the mass parse classic sniff db for:
```
SELECT *
FROM creature_unique_emote
WHERE entry=69
```

File: sql/migrations/20221106005233_world.sql [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+-- Events list for Young Wolf
+DELETE FROM `creature_ai_events` WHERE `creature_id`=299;
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (29901, 299, 0, 1, 0, 10, 1, 30000, 600000, 120000, 600000, 29901, 0, 0, 'Young Wolf - Howl and Emote OOC');
+
+DELETE FROM `creature_ai_scripts` WHERE `id`=29901;
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `priority`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(29901, 0, 0, 16, 1018, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Young Wolf - Play Sound 1018'),
```
Comment (ratkosrb) [inline sql/migrations/20221106005233_world.sql]: no such data in sniffs

File: sql/migrations/20221106005233_world.sql [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+(29901, 0, 0, 1, 15, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Young Wolf - Emote 15');
```
Comment (ratkosrb) [inline sql/migrations/20221106005233_world.sql]: no such data in sniffs

File: sql/migrations/20221106005233_world.sql [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+-- Events list for Coyote
+DELETE FROM `creature_ai_events` WHERE `creature_id`=834;
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (83401, 834, 0, 1, 0, 10, 1, 30000, 600000, 120000, 600000, 83401, 0, 0, 'Coyote - Howl and Emote OOC');
+
+DELETE FROM `creature_ai_scripts` WHERE `id`=83401;
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `priority`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(83401, 0, 0, 16, 1018, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Coyote - Play Sound 1018'),
```
Comment (ratkosrb) [inline sql/migrations/20221106005233_world.sql]: no such data in sniffs

File: sql/migrations/20221106005233_world.sql [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+(83401, 0, 0, 1, 15, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Coyote - Emote 15');
```
Comment (ratkosrb) [inline sql/migrations/20221106005233_world.sql]: no such data in sniffs

File: sql/migrations/20221106005233_world.sql [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+
+-- Events list for Young Scavenger
+DELETE FROM `creature_ai_events` WHERE `creature_id`=1508;
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (150801, 1508, 0, 1, 0, 10, 1, 30000, 600000, 120000, 600000, 150801, 0, 0, 'Young Scavenger - Howl and Emote OOC');
+
+DELETE FROM `creature_ai_scripts` WHERE `id`=150801;
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `priority`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(150801, 0, 0, 16, 1018, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Young Scavenger - Play Sound 1018'),
```
Comment (ratkosrb) [inline sql/migrations/20221106005233_world.sql]: no such data in sniffs

File: sql/migrations/20221106005233_world.sql [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+(150801, 0, 0, 1, 15, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Young Scavenger - Emote 15');
```
Comment (ratkosrb) [inline sql/migrations/20221106005233_world.sql]: no such data in sniffs

File: sql/migrations/20221106005233_world.sql [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+
+-- Events list for Ragged Scavenger
+DELETE FROM `creature_ai_events` WHERE `creature_id`=1509;
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (150901, 1509, 0, 1, 0, 10, 1, 30000, 600000, 120000, 600000, 150901, 0, 0, 'Ragged Scavenger - Howl and Emote OOC');
+
+DELETE FROM `creature_ai_scripts` WHERE `id`=150901;
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `priority`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(150901, 0, 0, 16, 1018, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Ragged Scavenger - Play Sound 1018'),
```
Comment (ratkosrb) [inline sql/migrations/20221106005233_world.sql]: no such data in sniffs

File: sql/migrations/20221106005233_world.sql [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+(150901, 0, 0, 1, 15, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Ragged Scavenger - Emote 15');
```
Comment (ratkosrb) [inline sql/migrations/20221106005233_world.sql]: no such data in sniffs

File: sql/migrations/20221106005233_world.sql [hunk 11]
```diff
@@ -0,0 +1,119 @@
...
+
+-- Events list for Gray Forest Wolf
+DELETE FROM `creature_ai_events` WHERE `creature_id`=1922;
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (192201, 1922, 0, 1, 0, 10, 1, 120000, 600000, 120000, 600000, 192201, 0, 0, 'Gray Forest Wolf - Howl and Emote OOC');
+
+DELETE FROM `creature_ai_scripts` WHERE `id`=192201;
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `priority`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(192201, 0, 0, 16, 1018, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Gray Forest Wolf - Play Sound 1018'),
```
Comment (ratkosrb) [inline sql/migrations/20221106005233_world.sql]: no such data in sniffs

File: sql/migrations/20221106005233_world.sql [hunk 12]
```diff
@@ -0,0 +1,119 @@
...
+(192201, 0, 0, 1, 15, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Gray Forest Wolf - Emote 15');
```
Comment (ratkosrb) [inline sql/migrations/20221106005233_world.sql]: no such data in sniffs

File: sql/migrations/20221106005233_world.sql [hunk 13]
```diff
@@ -0,0 +1,119 @@
...
+
+-- Events list for Elder Crag Coyote
+DELETE FROM `creature_ai_events` WHERE `creature_id`=2729;
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (272901, 2729, 0, 1, 0, 5, 1, 30000, 600000, 120000, 600000, 272901, 0, 0, 'Elder Crag Coyote - Howl and Emote OOC');
+
+DELETE FROM `creature_ai_scripts` WHERE `id`=272901;
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `priority`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(272901, 0, 0, 16, 1018, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Elder Crag Coyote - Play Sound 1018'),
```
Comment (ratkosrb) [inline sql/migrations/20221106005233_world.sql]: no such data in sniffs

File: sql/migrations/20221106005233_world.sql [hunk 14]
```diff
@@ -0,0 +1,119 @@
...
+(272901, 0, 0, 1, 15, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Elder Crag Coyote - Emote 15');
```
Comment (ratkosrb) [inline sql/migrations/20221106005233_world.sql]: no such data in sniffs

File: sql/migrations/20221106005233_world.sql [hunk 15]
```diff
@@ -0,0 +1,119 @@
...
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (295802, 2958, 0, 9, 0, 100, 1, 0, 5, 45000, 45000, 295802, 0, 0, 'Prairie Wolf - Cast Threatening Growl');
+
+DELETE FROM `creature_ai_scripts` WHERE `id`=295801;
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `priority`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(295801, 0, 0, 16, 1018, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Prairie Wolf - Play Sound 1018'),
```
Comment (ratkosrb) [inline sql/migrations/20221106005233_world.sql]: no such data in sniffs

File: sql/migrations/20221106005233_world.sql [hunk 16]
```diff
@@ -0,0 +1,119 @@
...
+(295801, 0, 0, 1, 15, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Prairie Wolf - Emote 15');
```
Comment (ratkosrb) [inline sql/migrations/20221106005233_world.sql]: no such data in sniffs

File: sql/migrations/20221106005233_world.sql [hunk 17]
```diff
@@ -0,0 +1,119 @@
...
+
+-- Events list for Prairie Stalker
+DELETE FROM `creature_ai_events` WHERE `creature_id`=2959;
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (295901, 2959, 0, 1, 0, 5, 1, 60000, 600000, 120000, 600000, 295901, 0, 0, 'Prairie Stalker - Howl and Emote OOC');
+
+DELETE FROM `creature_ai_scripts` WHERE `id`=295901;
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `priority`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(295901, 0, 0, 16, 1018, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Prairie Stalker - Play Sound 1018'),
```
Comment (ratkosrb) [inline sql/migrations/20221106005233_world.sql]: no such data in sniffs

File: sql/migrations/20221106005233_world.sql [hunk 18]
```diff
@@ -0,0 +1,119 @@
...
+(295901, 0, 0, 1, 15, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Prairie Stalker - Emote 15');
```
Comment (ratkosrb) [inline sql/migrations/20221106005233_world.sql]: no such data in sniffs

File: sql/migrations/20221106005233_world.sql [hunk 19]
```diff
@@ -0,0 +1,119 @@
...
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (296002, 2960, 0, 9, 0, 100, 1, 0, 5, 45000, 45000, 296002, 0, 0, 'Prairie Wolf Alpha - Cast Threatening Growl');
+
+DELETE FROM `creature_ai_scripts` WHERE `id`=296001;
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `priority`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(296001, 0, 0, 16, 1018, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Prairie Wolf Alpha - Play Sound 1018'),
```
Comment (ratkosrb) [inline sql/migrations/20221106005233_world.sql]: no such data in sniffs

File: sql/migrations/20221106005233_world.sql [hunk 20]
```diff
@@ -0,0 +1,119 @@
...
+(296001, 0, 0, 1, 15, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Prairie Wolf Alpha - Emote 15');
```
Comment (ratkosrb) [inline sql/migrations/20221106005233_world.sql]: no such data in sniffs

File: sql/migrations/20221106005233_world.sql [hunk 21]
```diff
@@ -0,0 +1,119 @@
...
+
+-- Events list for Longtooth Howler
+DELETE FROM `creature_ai_events` WHERE `creature_id`=5287;
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES (528701, 5287, 0, 2, 0, 100, 1, 20, 0, 0, 0, 528701, 0, 0, 'Longtooth Howler - Howl and Emote and Call For Help at 20% HP');
+
+DELETE FROM `creature_ai_scripts` WHERE `id`=528701;
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `priority`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(528701, 0, 0, 16, 1018, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Longtooth Howler - Play Sound 1018'),
```
Comment (ratkosrb) [inline sql/migrations/20221106005233_world.sql]: no such data in sniffs

File: sql/migrations/20221106005233_world.sql [hunk 22]
```diff
@@ -0,0 +1,119 @@
...
+(528701, 0, 0, 1, 15, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Longtooth Howler - Emote 15'),
```
Comment (ratkosrb) [inline sql/migrations/20221106005233_world.sql]: no such data in sniffs

File: sql/migrations/20221106005233_world.sql [hunk 23]
```diff
@@ -0,0 +1,119 @@
...
+(528701, 0, 0, 50, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 30, 0, 0, 0, 0, 'Longtooth Howler - Call for Help');
```
Comment (ratkosrb) [inline sql/migrations/20221106005233_world.sql:67]: Confirmed, but you forgot the text '%s calls for help!'.

**Comment by ratkosrb** [comment]
Literally everything in this pull request is wrong, except that Longtooth Howler should call for help.

**Comment by ratkosrb** [comment]
> I usually don't mind digging for evidence. But I am not going to subscribe to classic for this one :>

You can sniff and test things for free on the Classic PTR, which does not require a subscription, and is up all the time with level 60 character templates. And you could have also just checked the mass parse databases we have.

**Comment by FlagFlayer** [comment]
Actually as of right now, unless you already have a PTR account, you cannot join the Classic PTR for the first time unless you currently have a subscription. I saw you tell me about the PTR beforehand and when I checked it couldn't be done without a subscription to get on the PTR for the first time.

**Comment by ratkosrb** [comment]
Of course you must have made a PTR account at some point in the past 18 years since WoW came out. You activate it once while you are subscribed, and you have access to it forever.

**Comment by Gamemechanicwow** [comment]
What a mess I started here. I just wanted to adjust the flags 😅 

I agree. It does seem like the sound/emotes should never have been there in the first place, so I will update the PR to remove them instead of fixing them :>

**Comment by ratkosrb** [comment]
If you remove all AI events from a creature, remember to unset ai_name in creature_template too. EventAI should not be assigned if there are no events.

**Comment by Wall-core** [comment]
People have complained about a similar issue regarding hourly bells in major cities being too loud, saying that the sound was sourced at the object on classic instead of directly played. I'm not sure if a mix of vanilla, pservs and classic have people just hearing things.

**Comment by Kittnz** [comment]
> 

I did the hourly bells event and on classic or retail it just uses SMSG_PLAY_SOUND to play the sound. And thats what we are using now atm.

**Pull request merged**